synapse/models/transport.py

```diff
@@ -90,35 +304,39 @@ def getModelDefs(self):
                     ('issuer:name', ('ou:name', {}), {
                         'doc': 'The name of the org which issued the registration.'}),
                 )),
+
+                ('transport:land:vehicle:type:taxonomy', {}, ()),
+
                 ('transport:land:vehicle', {}, (
+
+                    ('type', ('transport:land:vehicle:type:taxonomy', {}), {
+                        'doc': 'A type taxonomy for land vehicles.'}),
+
+                    ('desc', ('str', {}), {
+                        'doc': 'A description of the vehicle.'}),
+
                     ('serial', ('str', {'strip': True}), {
                         'doc': 'The serial number or VIN of the vehicle.'}),
-                    ('built', ('time', {}), {
-                        'doc': 'The date the vehicle was constructed.'}),
+
                     ('make', ('ou:name', {}), {
-                        'doc': 'The make of the vehicle.'}),
-                    ('model', ('str', {'lower': True, 'onespace': True}), {
-                        'doc': 'The model of the vehicle.'}),
+                        'deprecated': True,
+                        'doc': 'Deprecated. Please use :manufacturer:name.'}),
+
                     ('registration', ('transport:land:registration', {}), {
                         'doc': 'The current vehicle registration information.'}),
-                    ('owner', ('ps:contact', {}), {
-                        'doc': 'The contact info of the owner of the vehicle.'}),
                 )),
                 ('transport:air:craft', {}, (
+
                     ('tailnum', ('transport:air:tailnum', {}), {
                         'doc': 'The aircraft tail number.'}),
+
+                    # TODO 3.x modify type to being a taxonomy.
```

Comment:
This has been updated to `transport:air:craft:type:taxonomy` in 3.x

Reply:
Thanks!